docker/.dockerignore

```diff
@@ -0,0 +1,11 @@
+# don't copy any of the python artifacts to the docker context
+__pycache__/
```

Comment:
This is just to ensure any Python bytecode, eggs, distribution data, or anything else isn't copied over, as well as the test suite, since we need to copy over an entire Python module and run it in separate steps, so we want to minimize the image size.